src/build.jl

```diff
@@ -0,0 +1,154 @@
+# This file generated the Markdown documentation files.
+
+# The @file macro generates the documentation for a particular file
+# where the {{method1, methods2}} includes the documentation for each method
+# via the `buildwriter` function.
+
+# Currently this prints the methodtable followed by the docstring.
+
+macro file(fn, md_string, m::Module) buildfile(fn, md_string, m) end
+
+function generate_for(m::Module, file_names...; doc_dir="docs", gen_dir="_generated")
+    eval(Expr(:toplevel, quote using $(symbol(m)) end))
```

Comment:
Is it necessary to import the module ``m`` like this? Shouldn't it already be imported if you're passing around a reference to it?